code/modules/food/recipes_microwave.dm

```diff
@@ -339,11 +339,17 @@
 /datum/recipe/poppypretzel
 	reagents = list(FLOUR = 5)
 	items = list(
-		/obj/item/seeds/poppyseed,
+		/obj/item/seeds,
 		/obj/item/weapon/reagent_containers/food/snacks/egg,
 		)
 	result = /obj/item/weapon/reagent_containers/food/snacks/poppypretzel
 
+/datum/recipe/poppypretzel/check_items(obj/container)
+	. = ..()
```

Comment:
not to nitpick or anything because i want to code better but can't you do a if(..()) or if(..() == 1) here?

Reply:
`check_items` can also return -1, so you would need to handle that case as well.

Reply:
what about if(..() == 1)?

Reply:
`check_items` can also return -1, so you would need to handle that case as well.

Reply:
what about if(..() == 1)?

Reply:
`check_items` can also return -1, so you would need to handle that case as well.

Reply:
what about if(..() == 1)?